app/experimenter/legacy-ui/templates/base.html

```diff
@@ -75,6 +75,27 @@ <h3 class="pt-2">
     </div>
   </div>
 
+  <div id="nimbus-cta">
+    <div class="container">
+      <div class="row py-3">
+        <div class="col-9">
+          <h5 class="font-bold">
+            <span class="fas fa-gift"></span>
+            Nimbus is now available!
+          </h5>
+          <p class="mb-0">You are encouraged to start using the all-new Nimbus Console for creating and evaluating experiments.</p>
+        </div>
+
+        <div class="col-3 text-right">
+          <a class="col btn btn-success" href="/nimbus" target="_blank" rel="noopener noreferrer">
```

Comment:
Q: Do we want `target` and `rel` here, or should we just swap the current page with the new Nimbus hotness (vs a new tab)?

Reply:
My thinking was since it's an entirely new application and could in theory take you away from a form you were working on I made it open in a new tab, but I'm certainly open to feedback!